examples/display_audio_photo/main/idf_component.yml

```diff
@@ -1,7 +1,7 @@
 description: BSP Display Audio Photo Example
 dependencies:
-  idf: ">=5.0"
+  idf: ">=4.4"
```

Comment:
I think you can completely remove `idf: ` line from `main/idf_component.yml`, since the example itself doesn't depend on any particular IDF version; It's enough to specify the dependency in the BSP component